nixos/tests/containers-imperative.nix

```diff
@@ -11,40 +11,11 @@ import ./make-test.nix ({ pkgs, ...} : {
     { imports = [ ../modules/installer/cd-dvd/channel.nix ];
       virtualisation.writableStore = true;
       virtualisation.memorySize = 768;
-
-      containers.webserver =
-        { privateNetwork = true;
-          hostAddress = "10.231.136.1";
-          localAddress = "10.231.136.2";
-          config =
-            { services.httpd.enable = true;
-              services.httpd.adminAddr = "foo@example.org";
-              networking.firewall.allowedTCPPorts = [ 80 ];
-              networking.firewall.allowPing = true;
-            };
-        };
-
       virtualisation.pathsInNixDB = [ pkgs.stdenv ];
     };
 
   testScript =
```

Comment:
@fpletz 
Hm, I re-ran these tests on three different machines (two ubuntu1404+nix, one nixos15.09) with cleaned nix-store but couldn't encounter this. Maybe the problem really was resolution of docbook.org at the time you ran the test?